2.6.md

```diff
@@ -855,7 +855,7 @@ This object describes the nature and behavior of the entity that is the source o
   <tr>
     <td><code>tid</code></td>
     <td>string; recommended</td>
-    <td>Transaction ID that must be common across all participants in this bid request (e.g., potentially multiple exchanges).</td>
+    <td>Transaction ID, generated by the publisher or agent originating an ad request for the end user, that must be common across all participants in this bid request (e.g., all exchanges participating in a header bidding or similar publisher-centric broadcast scenario). It should not be assumed that this ID uniquely identifies 1 biddable impression opportunity, unless that's all that exists on the bid request. Any podded audio/video bid request or multi-impression bid request may reflect multiple concurrent biddable impression opportunities. To uniquely identify those, consider using <a href="https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/per-imp-tids.md">per-impression transaction IDs</a>.</td>
```

Comment:
LGTM.

Reply:
I would consider adding: 

This should not be added by middlemen, with better wording. We've found some people don't support the tid on their request and then make one up when they make a request to the next entity.